filebeat/docs/reference/configuration/filebeat-options.asciidoc

```diff
@@ -319,6 +319,7 @@ Example configuration:
 [source,yaml]
 -------------------------------------------------------------------------------------
 json.message_key: log
+json.message_key: "json"
```

Comment:
`s/message_key/json_key/`

To be consistent with the line above which does not use quotes around `log`, drop the quotes from `"json"`.